lib/importmap/packager.rb

```diff
@@ -62,6 +63,19 @@ def remove(package)
     remove_package_from_importmap(package)
   end
 
+  def pin_options_for_package(package)
+    line = package_line_in_importmap(package) || ""
+    raw_options = line.match(/^#{base_package_line_regex(package)}?,[\s+]?(?<pin_options>.*) #.*$/)
```

Comment:
I noticed the dummy app’s `importmap.rb` and some of the `importmap.rb` fixtures do not include the comment with the version for the pins. I think his may be due to how vendored pinning used to output? The regex may need to be relaxed to capture these lines.